albinos-editor/src/albinospkg/cli.nim

```diff
@@ -0,0 +1,16 @@
+import terminal
+import rdstdin
+import linenoise
+
+func initCLI() {.inline.} =
+    discard historySetMaxLen(500)
+
+proc launchCLI*() =
+    initCLI()
+    while true:
```

Comment:
Will this properly quit on ctrl-D ?

Reply:
Ctrl-D is sigterm?

Reply:
I think it's supposed to cause sigterm internally. IIRC in C at least you're supposed to check by looking if `read` returns 0.